Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -200,5 +234,22 @@ click: function () { closeClicked.raiseEvent(this); }');
         return destroyObject(this);
     };
 
+    InfoBox.stringToHtml = function stringToHtml(s, frame) {
+        // if we don't have a script tag - in case the method was used as static...
+        if (s.indexOf('<script') > -1) {
```

Comment:
If we want to be thorough on security, it's a bit more complex... I believe the user should do sanitization. Otherwise, it would mean adding an external lib to Cesium (sanitization and escaping).  Writing our own sanitization (e.g. regexp) is risky and might add more vulnerabilities than solve them ([See slides 58 and on here](https://www.slideshare.net/LiranTal1/cluj-jsheroes-2017-liran-tal-on-nodejs-security). Do we want to go down that path?